src/odemis/driver/test/technolution_test.py

```diff
@@ -432,6 +432,58 @@ def test_resolution_VA(self):
         self.EBeamScanner.resolution.value = (6385, 6385)
         self.assertEqual(self.EBeamScanner.resolution.value, (6400, 6400))
 
+    def test_VA_dependencies(self):
+        """ Testing the dependencies between the scanner.resolution, mppc.cellCompleteResolution and
+        mppc.cellTranslation VAs. """
+
+        # rule: max cell translation <= cell complete size - effective cell size
+        self.MPPC.cellTranslation.value = \
+            tuple(tuple((0, 0) for i in range(0, self.MPPC.shape[0])) for i in range(0, self.MPPC.shape[1]))
+        self.EBeamScanner.resolution.value = (6400, 6400)  # -> cell eff res = 800
+        self.MPPC.cellCompleteResolution.value = (810, 810)
+        with self.assertRaises(ValueError):
+            # not max cell translation <= cell complete size - effective cell size  (! 50 <= 810 - 800)
+            self.MPPC.cellTranslation.value = \
+                tuple(tuple((50, 50) for i in range(0, self.MPPC.shape[0])) for i in range(0, self.MPPC.shape[1]))
```

Comment:
Please add a test case which fails with the current max(max()) implementation. I think it's just a matter of pass good X values, but incorrect Y values.